src/api/schema/metrics/sent_events.rs

```diff
@@ -72,6 +83,10 @@ impl ComponentSentEventsTotal {
         self.component_key.id()
     }
 
+    async fn outputs(&self) -> Vec<Output> {
+        self.outputs.clone()
+    }
```

Comment:
No need to clone here:

```suggestion
    async fn outputs(&self) -> Vec<&Output> {
        &self.outputs
    }
```